retroshare-nogui/src/retroshare-nogui.pro

```diff
@@ -58,6 +58,9 @@ linux-g++-64 {
 	OBJECTS_DIR = temp/linux-g++-64/obj
 }
 
+rs_sanitize {
+	LIBS *= -lasan -lubsan
+}
```

Comment:
This should probably go in `retroshare.pri` otherwise when `rs_sanitize` is enabled for `retroshare-service` linking will probably fail